service/src/main/java/com/epam/esm/service/impl/GiftCertificateServiceImpl.java

```diff
@@ -47,6 +49,11 @@ public GiftCertificateDto create(GiftCertificateInputDto giftCertificateInputDto
         );
     }
 
+    @Override
+    public GiftCertificateDto create(GiftCertificateDto value) {
+        return null;
```

Comment:
Please refactor this method: the motivation of returning `null` is unclear here.